src/main/java/gregtech/api/metatileentity/multiblock/DummyCleanroom.java

```diff
@@ -0,0 +1,53 @@
+package gregtech.api.metatileentity.multiblock;
+
+import javax.annotation.Nonnull;
+import java.util.Collection;
+import java.util.Collections;
+
+public final class DummyCleanroom implements ICleanroomProvider {
+
+    private final boolean allowsAllTypes;
+    private final Collection<CleanroomType> allowedTypes;
+
+    public DummyCleanroom(@Nonnull Collection<CleanroomType> allowedTypes) {
+        this.allowsAllTypes = false;
+        this.allowedTypes = allowedTypes;
+    }
+
+    public DummyCleanroom(boolean allowsAllTypes) {
+        this.allowsAllTypes = allowsAllTypes;
+        this.allowedTypes = Collections.emptyList();
+    }
```

Comment:
Minor: does this need a boolean param? Passing false here doesn't seem useful (and even if it is, you can pass it in the above ctor with `Collections.emptyList()`